gwpy/cli/cliproduct.py

```diff
@@ -58,6 +58,7 @@ class CliProduct(object):
     """
 
     __metaclass__ = abc.ABCMeta
+    BAD_UNITS = ['undef', '', '*']
```

Comment:
@areeda, can you change this to just

```python
    BAD_UNITS = {'*',}
```

since the other two are legitimate unit choices? (also, `a in {set}` is slightly more efficient than `a in {list}` (but only really for large lists).

Reply:
these are in latest push